mrmustard/physics/ansatze.py

```diff
@@ -530,3 +530,51 @@ def conj(self):
         The conjugate of this ansatz.
         """
         return self.__class__(math.conj(self.array))
+
+
+def Abc_to_cov_and_mean(A: Matrix, b: Vector, c: Scalar) -> Union[Matrix, Vector, Scalar]:
```

Comment:
I would put all these functions in `triples.py` or in `converters.py`. IMO, they do not belong to `ansatze.py`

Reply:
I delete this function to open an new PR!